contrib/docker-compose/docker-compose.yml

```diff
@@ -45,3 +45,4 @@ services:
 volumes:
   libvirtd: {}
   libvirt_qemu: {}
+  run: {}
```

Comment:
After this change you can also remove (imo this should done in same commit) [this file](https://github.com/Mirantis/virtlet/blob/b360335833c922e2642f275446da0dc722a9b132/contrib/docker-compose/run/empty.txt).